From 1dad64eb1838aceb353b48fa6d9624fe84ba8f89 Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Wed, 22 Aug 2018 16:28:31 +0000
Subject: [PATCH 3/8] add AF_XDP to sys/socket.h from linux v4.18

new address family and related macros were added in linux commit
68e8b849b221b37a78a110a0307717d45e3593a0
---
 include/sys/socket.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/include/sys/socket.h b/include/sys/socket.h
index 14be6e99..1dec4b00 100644
--- a/include/sys/socket.h
+++ b/include/sys/socket.h
@@ -109,7 +109,8 @@ struct linger {
 #define PF_KCM          41
 #define PF_QIPCRTR      42
 #define PF_SMC          43
-#define PF_MAX          44
+#define PF_XDP          44
+#define PF_MAX          45
 
 #define AF_UNSPEC       PF_UNSPEC
 #define AF_LOCAL        PF_LOCAL
@@ -158,6 +159,7 @@ struct linger {
 #define AF_KCM          PF_KCM
 #define AF_QIPCRTR      PF_QIPCRTR
 #define AF_SMC          PF_SMC
+#define AF_XDP          PF_XDP
 #define AF_MAX          PF_MAX
 
 #ifndef SO_DEBUG
@@ -265,6 +267,7 @@ struct linger {
 #define SOL_NFC         280
 #define SOL_KCM         281
 #define SOL_TLS         282
+#define SOL_XDP         283
 
 #define SOMAXCONN       128
 
-- 
2.17.1

